src/CNLoginServer.cpp

```diff
@@ -422,11 +429,19 @@ bool CNLoginServer::exitDuplicate(int accountId) {
 
 bool CNLoginServer::isLoginDataGood(std::string login, std::string password) {
     std::regex loginRegex("[a-zA-Z0-9_-]{4,32}");
+    //web api sends password hashed, so we don't check it
+    if (settings::USEWEBAPI)
```

Comment:
After an internal discussion, we're going to wait to merge anything relating to the web API frontend until later. In an ideal situation, we should be sending passwords over TLS from the client to the server however that's out of scope for the time being. Please remove this option for now. Sorry for not reaching out sooner!